source/common/api/api_impl.h

```diff
@@ -18,7 +16,8 @@ namespace Api {
  */
 class Impl : public Api {
 public:
-  Impl(Thread::ThreadFactory& thread_factory, Stats::Store&, Event::TimeSystem& time_system);
+  Impl(Thread::ThreadFactory& thread_factory, Stats::Store&, Event::TimeSystem& time_system,
+       Filesystem::Instance& file_system_);
```

Comment:
Same here, local argument shouldn't end in `_`

Reply:
Done.